### PR TITLE
Enable "Linq to foreach" refactoring in VS

### PR DESCRIPTION
**Pull request by ivanbasov**
VSO: [Bug 602003: linq to foreach refactoring is not enabled in VS](https://devdiv.visualstudio.com/DevDiv/_queries?id=602003)

### CODE REVIEW

File: src/Features/CSharp/Portable/ConvertLinq/CSharpConvertLinqQueryToForEachProvider.cs [hunk 1]
```diff
@@ -13,12 +15,12 @@
 using Microsoft.CodeAnalysis.LanguageServices;
 using Microsoft.CodeAnalysis.Operations;
 using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Shared.Utilities;
 using Microsoft.CodeAnalysis.Simplification;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CSharp.ConvertLinq
 {
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(CSharpConvertLinqQueryToForEachProvider)), Shared]
```
Comment (mavasani) [inline src/Features/CSharp/Portable/ConvertLinq/CSharpConvertLinqQueryToForEachProvider.cs]: We should consider writing a meta analyzer in Microsoft.CodeAnalysis.Analyzers to catch this. Can you please file a bug in analyzers repo?

Comment (mavasani) [reply src/Features/CSharp/Portable/ConvertLinq/CSharpConvertLinqQueryToForEachProvider.cs]: Note we already have such an analyzer/fixer for diagnostic analyzers that don't have `DiagnosticAnalyzerAttribute`.

Comment (ivanbasov) [reply src/Features/CSharp/Portable/ConvertLinq/CSharpConvertLinqQueryToForEachProvider.cs]: Thank you! Created https://github.com/dotnet/roslyn-analyzers/issues/1670

**Comment by jinujoseph** [comment]
approved to merge via [link](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/602003) for 15.7.Preview5

**Pull request merged**